src/hpa.c

```diff
@@ -247,12 +247,20 @@ hpa_try_alloc_no_grow(tsdn_t *tsdn, hpa_shard_t *shard, size_t size, bool *oom)
 	}
 	assert(edata_arena_ind_get(edata) == shard->ind);
 
-	err = psset_alloc_reuse(&shard->psset, edata, size);
-	if (err) {
+	hpdata_t *ps = psset_fit(&shard->psset, size);
```

Comment:
Shall we call out _oldest_fit?

Reply:
My inclination is no:
- This matches the eset naming
- The fragmentation avoidance policy is an internal implementation choice of the psset (and in fact, it isn't quite oldest fit -- it's the oldest fit of the best fits. And even there, I think we'll start doing some degree of hugepage-preference).
- Eventually, we'll have a psset function for picking hpdatas to hugify or purge, and we'll want some consistent naming scheme for the various ways to pick things we have.